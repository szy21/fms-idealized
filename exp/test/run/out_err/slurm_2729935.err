echo *** Running /home/zhaoyi/fms-idealized/exp/test/run/run_test_1x_hpc on hpc-22-14 ***
date
set analysis_version = analysis_3d
set analysis_script = run_analysis_moist_3d_hpc
set diagtable = /home/zhaoyi/fms-idealized/exp/test/input/diag_table_moist_3d
set analysis_dir = /home/zhaoyi/fms-idealized/analysis/analysis_3d/run
source /etc/profile.d/modules.csh
if ( 1 ) then
set modules_shell=tcsh
else
set exec_prefix=/software/Modules/bin
set prefix=
set postfix=
if ( 0 ) then
set histchar = !
endif
if ( 0 ) then
if ( 0 ) then
set postfix = set _exit=$status; ; /usr/bin/test 0 = $_exit;
alias module eval `/software/Modules/bin/modulecmd tcsh !*`; set _exit=$status; ; /usr/bin/test 0 = $_exit;
unset exec_prefix
unset prefix
unset postfix
setenv MODULESHOME /software/Modules
if ( ! 1 ) then
if ( ! 1 ) then
eval `/software/Modules/bin/modulecmd tcsh load hdf5/1.10.2`
/software/Modules/bin/modulecmd tcsh load hdf5/1.10.2
setenv CPATH /software/hdf5/1.10.1/include
setenv HDF5_DIR /software/hdf5/1.10.1
setenv LD_LIBRARY_PATH /software/hdf5/1.10.1/lib
setenv LIBRARY_PATH /software/hdf5/1.10.1/lib
setenv LOADEDMODULES hdf5/1.10.2
setenv MANPATH /software/hdf5/1.10.1/share/man:/central/slurm/install/current/share/man/:/usr/local/share/man:/usr/share/man/overrides:/usr/share/man:/opt/ibutils/share/man:/cm/local/apps/environment-modules/current/share/man
setenv PATH /software/hdf5/1.10.1/bin:/home/zhaoyi/local/bin:/central/slurm/install/current/bin/:/usr/lib64/qt-3.3/bin:/usr/local/bin:/usr/bin:/usr/local/sbin:/usr/sbin:/usr/lpp/mmfs/bin:/opt/ibutils/bin:/opt/dell/srvadmin/bin:/home/zhaoyi/.local/bin:/home/zhaoyi/bin
setenv _LMFILES_ /software/Modules/modulefiles/data/hdf5/1.10.2
set _exit=0
/usr/bin/test 0 = 0
eval `/software/Modules/bin/modulecmd tcsh load netcdf-c/4.6.1`
/software/Modules/bin/modulecmd tcsh load netcdf-c/4.6.1
setenv CPATH /central/software/netcdf-c/4.6.1//include:/software/hdf5/1.10.1/include
setenv LD_LIBRARY_PATH /central/software/netcdf-c/4.6.1//lib:/software/hdf5/1.10.1/lib
setenv LD_RUN_PATH /central/software/netcdf-c/4.6.1//lib
setenv LIBRARY_PATH /central/software/netcdf-c/4.6.1//lib:/software/hdf5/1.10.1/lib
setenv LOADEDMODULES hdf5/1.10.2:netcdf-c/4.6.1
setenv MANPATH /central/software/netcdf-c/4.6.1//share/man:/software/hdf5/1.10.1/share/man:/central/slurm/install/current/share/man/:/usr/local/share/man:/usr/share/man/overrides:/usr/share/man:/opt/ibutils/share/man:/cm/local/apps/environment-modules/current/share/man
setenv PATH /central/software/netcdf-c/4.6.1//bin:/software/hdf5/1.10.1/bin:/home/zhaoyi/local/bin:/central/slurm/install/current/bin/:/usr/lib64/qt-3.3/bin:/usr/local/bin:/usr/bin:/usr/local/sbin:/usr/sbin:/usr/lpp/mmfs/bin:/opt/ibutils/bin:/opt/dell/srvadmin/bin:/home/zhaoyi/.local/bin:/home/zhaoyi/bin
setenv _LMFILES_ /software/Modules/modulefiles/data/hdf5/1.10.2:/software/Modules/modulefiles/data/netcdf-c/4.6.1
set _exit=0
/usr/bin/test 0 = 0
eval `/software/Modules/bin/modulecmd tcsh load netcdf-fortran/4.4.4`
/software/Modules/bin/modulecmd tcsh load netcdf-fortran/4.4.4
setenv CPATH /central/software/netcdf-fortran/4.4.4/include:/central/software/netcdf-c/4.6.1//include:/software/hdf5/1.10.1/include
setenv LD_LIBRARY_PATH /central/software/netcdf-fortran/4.4.4/lib:/central/software/netcdf-c/4.6.1//lib:/software/hdf5/1.10.1/lib
setenv LD_RUN_PATH /central/software/netcdf-fortran/4.4.4/lib:/central/software/netcdf-c/4.6.1//lib
setenv LIBRARY_PATH /central/software/netcdf-fortran/4.4.4/lib:/central/software/netcdf-c/4.6.1//lib:/software/hdf5/1.10.1/lib
setenv LOADEDMODULES hdf5/1.10.2:netcdf-c/4.6.1:netcdf-fortran/4.4.4
setenv MANPATH /central/software/netcdf-fortran/4.4.4/share/man:/central/software/netcdf-c/4.6.1//share/man:/software/hdf5/1.10.1/share/man:/central/slurm/install/current/share/man/:/usr/local/share/man:/usr/share/man/overrides:/usr/share/man:/opt/ibutils/share/man:/cm/local/apps/environment-modules/current/share/man
setenv PATH /central/software/netcdf-fortran/4.4.4/bin:/central/software/netcdf-c/4.6.1//bin:/software/hdf5/1.10.1/bin:/home/zhaoyi/local/bin:/central/slurm/install/current/bin/:/usr/lib64/qt-3.3/bin:/usr/local/bin:/usr/bin:/usr/local/sbin:/usr/sbin:/usr/lpp/mmfs/bin:/opt/ibutils/bin:/opt/dell/srvadmin/bin:/home/zhaoyi/.local/bin:/home/zhaoyi/bin
setenv _LMFILES_ /software/Modules/modulefiles/data/hdf5/1.10.2:/software/Modules/modulefiles/data/netcdf-c/4.6.1:/software/Modules/modulefiles/data/netcdf-fortran/4.4.4
set _exit=0
/usr/bin/test 0 = 0
eval `/software/Modules/bin/modulecmd tcsh load ncview`
/software/Modules/bin/modulecmd tcsh load ncview
ModuleCmd_Load.c(213):ERROR:105: Unable to locate a modulefile for 'ncview'
set _exit=0
/usr/bin/test 0 = 0
echo MPI Used: `which srun`
which srun
eval `/software/Modules/bin/modulecmd tcsh load intel/18.1`
/software/Modules/bin/modulecmd tcsh load intel/18.1
setenv CC icc
setenv CLASSPATH /central/software/Intel/2018.1/compilers_and_libraries_2018.1.163/linux/mpi/intel64/lib/mpi.jar:/central/software/Intel/2018.1/compilers_and_libraries_2018.1.163/linux/daal/lib/daal.jar
setenv CPATH /central/software/Intel/2018.1/compilers_and_libraries_2018.1.163/linux/ipp/include:/central/software/Intel/2018.1/compilers_and_libraries_2018.1.163/linux/mkl/include:/central/software/Intel/2018.1/compilers_and_libraries_2018.1.163/linux/pstl/include:/central/software/Intel/2018.1/compilers_and_libraries_2018.1.163/linux/tbb/include:/central/software/Intel/2018.1/compilers_and_libraries_2018.1.163/linux/tbb/include:/central/software/Intel/2018.1/compilers_and_libraries_2018.1.163/linux/daal/include:/central/software/netcdf-fortran/4.4.4/include:/central/software/netcdf-c/4.6.1//include:/software/hdf5/1.10.1/include
setenv CXX icpc
setenv DAALROOT /central/software/Intel/2018.1/compilers_and_libraries_2018.1.163/linux/daal
setenv F77 ifort
setenv F90 ifort
setenv GDBSERVER_MIC /central/software/Intel/2018.1/debugger_2018/gdb/targets/intel64/x200/bin/gdbserver
setenv GDB_CROSS /central/software/Intel/2018.1/debugger_2018/gdb/intel64/bin/gdb-ia
setenv INFOPATH /central/software/Intel/2018.1/documentation_2018/en/debugger//gdb-ia/info/:/central/software/Intel/2018.1/documentation_2018/en/debugger//gdb-igfx/info/
setenv INTEL_HOME /central/software/Intel/2018.1
setenv INTEL_LICENSE_FILE /central/software/Intel/2018.1/compilers_and_libraries_2018.1.163/linux/licenses:/opt/intel/licenses
setenv INTEL_PYTHONHOME /central/software/Intel/2018.1/debugger_2018/python/intel64/
setenv INTEL_VERSION 18.1
setenv IPPROOT /central/software/Intel/2018.1/compilers_and_libraries_2018.1.163/linux/ipp
setenv I_MPI_CC icc
setenv I_MPI_CXX icpc
setenv I_MPI_F77 ifort
setenv I_MPI_F90 ifort
setenv I_MPI_ROOT /central/software/Intel/2018.1/compilers_and_libraries_2018.1.163/linux/mpi
setenv LD_LIBRARY_PATH /central/software/Intel/2018.1/compilers_and_libraries_2018.1.163/linux/compiler/lib/intel64:/central/software/Intel/2018.1/compilers_and_libraries_2018.1.163/linux/compiler/lib/intel64_lin:/central/software/Intel/2018.1/compilers_and_libraries_2018.1.163/linux/mpi/intel64/lib:/central/software/Intel/2018.1/compilers_and_libraries_2018.1.163/linux/mpi/mic/lib:/central/software/Intel/2018.1/compilers_and_libraries_2018.1.163/linux/ipp/lib/intel64:/central/software/Intel/2018.1/compilers_and_libraries_2018.1.163/linux/mkl/lib/intel64_lin:/central/software/Intel/2018.1/compilers_and_libraries_2018.1.163/linux/tbb/lib/intel64/gcc4.7:/central/software/Intel/2018.1/debugger_2018/iga/lib:/central/software/Intel/2018.1/debugger_2018/libipt/intel64/lib:/central/software/Intel/2018.1/compilers_and_libraries_2018.1.163/linux/daal/lib/intel64_lin:/central/software/Intel/2018.1/compilers_and_libraries_2018.1.163/linux/daal/../tbb/lib/intel64_lin/gcc4.4:/central/software/netcdf-fortran/4.4.4/lib:/central/software/netcdf-c/4.6.1//lib:/software/hdf5/1.10.1/lib
setenv LIBRARY_PATH /central/software/Intel/2018.1/compilers_and_libraries_2018.1.163/linux/ipp/lib/intel64:/central/software/Intel/2018.1/compilers_and_libraries_2018.1.163/linux/compiler/lib/intel64_lin:/central/software/Intel/2018.1/compilers_and_libraries_2018.1.163/linux/mkl/lib/intel64_lin:/central/software/Intel/2018.1/compilers_and_libraries_2018.1.163/linux/tbb/lib/intel64/gcc4.7:/central/software/Intel/2018.1/compilers_and_libraries_2018.1.163/linux/daal/lib/intel64_lin:/central/software/Intel/2018.1/compilers_and_libraries_2018.1.163/linux/daal/../tbb/lib/intel64_lin/gcc4.4:/central/software/netcdf-fortran/4.4.4/lib:/central/software/netcdf-c/4.6.1//lib:/software/hdf5/1.10.1/lib
setenv LOADEDMODULES hdf5/1.10.2:netcdf-c/4.6.1:netcdf-fortran/4.4.4:intel/18.1
setenv MANPATH /central/software/Intel/2018.1/man/common:/central/software/Intel/2018.1/compilers_and_libraries_2018.1.163/linux/mpi/man:/central/software/Intel/2018.1/documentation_2018/en/debugger//gdb-ia/man/:/central/software/Intel/2018.1/documentation_2018/en/debugger//gdb-igfx/man/:/central/software/netcdf-fortran/4.4.4/share/man:/central/software/netcdf-c/4.6.1//share/man:/software/hdf5/1.10.1/share/man:/central/slurm/install/current/share/man/:/usr/local/share/man:/usr/share/man/overrides:/usr/share/man:/opt/ibutils/share/man:/cm/local/apps/environment-modules/current/share/man
setenv MKLROOT /central/software/Intel/2018.1/compilers_and_libraries_2018.1.163/linux/mkl
setenv MPM_LAUNCHER /central/software/Intel/2018.1/debugger_2018/mpm/mic/bin/start_mpm.sh
setenv NLSPATH /central/software/Intel/2018.1/compilers_and_libraries_2018.1.163/linux/compiler/lib/intel64/locale/%l_%t/%N:/central/software/Intel/2018.1/compilers_and_libraries_2018.1.163/linux/mkl/lib/intel64_lin/locale/%l_%t/%N:/central/software/Intel/2018.1/debugger_2018/gdb/intel64/share/locale/%l_%t/%N
setenv PATH /central/software/Intel/2018.1/compilers_and_libraries_2018.1.163/linux/bin/intel64:/central/software/Intel/2018.1/compilers_and_libraries_2018.1.163/linux/mpi/intel64/bin:/central/software/netcdf-fortran/4.4.4/bin:/central/software/netcdf-c/4.6.1//bin:/software/hdf5/1.10.1/bin:/home/zhaoyi/local/bin:/central/slurm/install/current/bin/:/usr/lib64/qt-3.3/bin:/usr/local/bin:/usr/bin:/usr/local/sbin:/usr/sbin:/usr/lpp/mmfs/bin:/opt/ibutils/bin:/opt/dell/srvadmin/bin:/home/zhaoyi/.local/bin:/home/zhaoyi/bin
setenv PKG_CONFIG_PATH /central/software/Intel/2018.1/compilers_and_libraries_2018.1.163/linux/mkl/bin/pkgconfig
setenv PSTLROOT /central/software/Intel/2018.1/compilers_and_libraries_2018.1.163/linux/pstl
setenv TBBROOT /central/software/Intel/2018.1/compilers_and_libraries_2018.1.163/linux/tbb
setenv _LMFILES_ /software/Modules/modulefiles/data/hdf5/1.10.2:/software/Modules/modulefiles/data/netcdf-c/4.6.1:/software/Modules/modulefiles/data/netcdf-fortran/4.4.4:/software/Modules/modulefiles/compilers/intel/18.1
set _exit=0
/usr/bin/test 0 = 0
limit stacksize unlimited
cd /home/zhaoyi/fms-idealized/exp/test
set tmpdir = /central/groups/esm/zhaoyi//fms_tmp/test
set run_dir = /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc
set workdir = /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc/workdir
set output_dir = /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc/output
set platform = ifc
set init_cond = 
set pathnames = /home/zhaoyi/fms-idealized/exp/test/input/path_names
set namelist = /home/zhaoyi/fms-idealized/exp/test/input/namelists_moist
set fieldtable = /home/zhaoyi/fms-idealized/exp/test/input/field_table_moist
set execdir = /central/groups/esm/zhaoyi//fms_tmp/test/exe.fms
set run_analysis = /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc/analysis
set analysis_out_err = /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc/analysis/out_err
set mppnccombine = /central/groups/esm/zhaoyi//fms_tmp/test/mppnccombine.ifc
set template = /home/zhaoyi/fms-idealized/idealized/bin/mkmf.template.ifc_hpc_mpi
set mkmf = /home/zhaoyi/fms-idealized/idealized/bin/mkmf
set sourcedir = /home/zhaoyi/fms-idealized/idealized/src
set time_stamp = /home/zhaoyi/fms-idealized/idealized/bin/time_stamp.csh
set ireload = 1
set irun = 1
set reload_file = /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc/reload_commands
if ( -d /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc ) then
if ( -f /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc/reload_commands ) then
endif
if ( ! -d /home/zhaoyi/fms-idealized/exp/test/run/out_err ) mkdir -p /home/zhaoyi/fms-idealized/exp/test/run/out_err
if ( ! -d /central/groups/esm/zhaoyi//fms_tmp/test/exe.fms ) mkdir -p /central/groups/esm/zhaoyi//fms_tmp/test/exe.fms
if ( ! -d /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc/analysis ) mkdir -p /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc/analysis
if ( ! -d /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc/analysis/out_err ) mkdir -p /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc/analysis/out_err
if ( ! -e /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc/workdir ) then
mkdir /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc/workdir /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc/workdir/INPUT /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc/workdir/RESTART
else
if ( ! -d /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc/output ) then
if ( ! -f /central/groups/esm/zhaoyi//fms_tmp/test/mppnccombine.ifc ) then
/bin/cp /home/zhaoyi/fms-idealized/idealized/bin/mkmf.template.ifc_hpc_mpi /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc/workdir/tmp_template
echo fms_home = /home/zhaoyi/fms-idealized/idealized
cd /home/zhaoyi/fms-idealized/idealized/src
find /home/zhaoyi/fms-idealized/exp/test/srcmods/ -maxdepth 1 -iname *.f90 -o -iname *.inc -o -iname *.c -o -iname *.h
echo Using the following sourcecode modifications:
cat /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc/workdir/tmp_pathnames
cat /home/zhaoyi/fms-idealized/exp/test/input/path_names
cd /central/groups/esm/zhaoyi//fms_tmp/test/exe.fms
/home/zhaoyi/fms-idealized/idealized/bin/mkmf -p fms.x -t /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc/workdir/tmp_template -c -Duse_libMPI -Duse_netCDF -a /home/zhaoyi/fms-idealized/idealized/src /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc/workdir/tmp_pathnames /home/zhaoyi/fms-idealized/idealized/src/shared/include /home/zhaoyi/fms-idealized/idealized/src/shared/mpp/include
........................................................................make -f Makefile
cd /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc/workdir/INPUT
if ( !=  ) then
set ocean_mask = ocean_mask_T42.nc
if ( -e /home/zhaoyi/fms-idealized/exp/test/input/ocean_mask_T42.nc ) then
while ( 1 < = 1 )
cd /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc/workdir
if ( 1 == 1 && 1 == 1 ) then
cat
else
if ( moist == dry ) then
if ( moist == moist ) then
cat
else if ( moist == moist_hydro ) then
cat
endif
cat /home/zhaoyi/fms-idealized/exp/test/input/namelists_moist
cp /home/zhaoyi/fms-idealized/exp/test/input/diag_table_moist_3d diag_table
cp /home/zhaoyi/fms-idealized/exp/test/input/field_table_moist field_table
cp /central/groups/esm/zhaoyi//fms_tmp/test/exe.fms/fms.x fms.x
cp input.nml /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc
set MX_RCACHE=2
srun /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc/workdir/fms.x

WARNING from PE    0: Namelist mix_snapshot_average_fields = true will cause ERROR in time coordinates of all time_averaged fields. Strongly recommend mix_snapshot_average_fields = false


WARNING from PE    0: register_static_field: module/field dynamics/pk is STATIC, can NOT time ops


WARNING from PE    0: register_static_field: module/field dynamics/bk is STATIC, can NOT time ops


WARNING from PE    0: register_static_field: module/field dynamics/zsurf is STATIC, can NOT time ops


WARNING from PE    1: mixed_layer: mixed_layer restart file not found


WARNING from PE    4: mixed_layer: mixed_layer restart file not found


WARNING from PE    5: mixed_layer: mixed_layer restart file not found


WARNING from PE    6: mixed_layer: mixed_layer restart file not found


WARNING from PE    7: mixed_layer: mixed_layer restart file not found


WARNING from PE    0: mixed_layer: mixed_layer restart file not found


WARNING from PE    3: mixed_layer: mixed_layer restart file not found


WARNING from PE    2: mixed_layer: mixed_layer restart file not found

set date_name = `$time_stamp -eh`
/home/zhaoyi/fms-idealized/idealized/bin/time_stamp.csh -eh
if ( day0010h00 ==  ) set date_name = tmp`date '+%j%H%M%S'`
if ( -f time_stamp.out ) rm -f time_stamp.out
rm -f time_stamp.out
mkdir /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc/output/combine/day0010h00
mkdir: cannot create directory ‘/central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc/output/combine/day0010h00’: File exists
foreach ncfile ( `/bin/ls *.nc *.nc.????` )
/bin/ls 1xday.nc.0000 1xday.nc.0001 1xday.nc.0002 1xday.nc.0003 1xday.nc.0004 1xday.nc.0005 1xday.nc.0006 1xday.nc.0007
mv 1xday.nc.0000 /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc/output/combine/day0010h00/day0010h00.1xday.nc.0000
end
mv 1xday.nc.0001 /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc/output/combine/day0010h00/day0010h00.1xday.nc.0001
end
mv 1xday.nc.0002 /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc/output/combine/day0010h00/day0010h00.1xday.nc.0002
end
mv 1xday.nc.0003 /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc/output/combine/day0010h00/day0010h00.1xday.nc.0003
end
mv 1xday.nc.0004 /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc/output/combine/day0010h00/day0010h00.1xday.nc.0004
end
mv 1xday.nc.0005 /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc/output/combine/day0010h00/day0010h00.1xday.nc.0005
end
mv 1xday.nc.0006 /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc/output/combine/day0010h00/day0010h00.1xday.nc.0006
end
mv 1xday.nc.0007 /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc/output/combine/day0010h00/day0010h00.1xday.nc.0007
end
foreach out ( `/bin/ls *.out` )
/bin/ls logfile.0000.out
mv logfile.0000.out /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc/output/logfiles/day0010h00.logfile.0000.out
end
cd /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc/workdir/RESTART
set resfiles = `/bin/ls *.res*`
/bin/ls atmos_model.res atmosphere.res.nc mixed_layer.res.nc spectral_dynamics.res.nc
if ( 4 > 0 ) then
set restart_file = /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc/output/restart/day0010h00.cpio
if ( ! -d /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc/output/restart ) mkdir -p /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc/output/restart
cp /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc/workdir/input.nml /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc/workdir/diag_table .
set files = ( atmos_model.res atmosphere.res.nc mixed_layer.res.nc spectral_dynamics.res.nc input.nml diag_table )
/bin/ls atmos_model.res atmosphere.res.nc mixed_layer.res.nc spectral_dynamics.res.nc input.nml diag_table
cpio -ocv
atmos_model.res
atmosphere.res.nc
diag_table
input.nml
mixed_layer.res.nc
spectral_dynamics.res.nc
33020 blocks
mv day0010h00.cpio /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc/output/restart/day0010h00.cpio
if ( 1 < 1 ) then
endif
cd /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc/workdir
set run_number = `expr $ireload \* $runs_per_script - $runs_per_script + $irun`
expr 1 * 1 - 1 + 1
echo Completed run 1 of 1 in bsub 1.
set irun_prev = 1
@ irun++
if ( -f /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc/reload_commands_prev ) then
if ( -f /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc/reload_commands ) mv -f /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc/reload_commands /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc/reload_commands_prev
if ( 2 < = 1 ) then
@ ireload++
echo set irun         =  1
endif
echo set init_cond    =  /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc/output/restart/day0010h00.cpio
echo set ireload      =  2
cd /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc/analysis
if ( 1 > = 1 ) then
set postproc_dir = /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc/analysis/day0010h00
if ( ! -e /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc/analysis/day0010h00 ) then
rm -rf /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc/analysis/day0010h00
mkdir /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc/analysis/day0010h00
echo WARNING: Existing analysis directory /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc/analysis/day0010h00 removed.
endif
cd /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc/analysis/day0010h00
echo set exp_name = test
echo set data_dir = /central/groups/esm/zhaoyi//fms_output/test/test_1x_hpc
echo set run_name = test_1x_hpc
echo set run_script = /home/zhaoyi/fms-idealized/exp/test/run/run_test_1x_hpc
echo set date_name = day0010h00
echo set run_analysis = /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc/analysis
echo set fms_home = /home/zhaoyi/fms-idealized/idealized
echo set tmpdir = /central/groups/esm/zhaoyi//fms_tmp/test
echo set `grep num_fourier /home/zhaoyi/fms-idealized/exp/test/input/namelists_moist | tr ',' ' ' `
tr ,  
grep num_fourier /home/zhaoyi/fms-idealized/exp/test/input/namelists_moist
echo set irun = 1
echo set runs_per_script = 1
echo set days_per_segment = 1
echo set num_segments = 10
echo set isegment = 1
echo set fms_output_freq = 1xday
cp /home/zhaoyi/fms-idealized/analysis/analysis_3d/run/run_analysis_moist_3d_hpc ./
cd /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc/analysis/day0010h00
sbatch run_analysis_moist_3d_hpc
else
if ( ! -f /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc/output/restart/day0010h00.cpio ) then
end
while ( 2 < = 1 )
rm -rf /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc/workdir
cd /home/zhaoyi/fms-idealized/exp/test/run
if ( 2 > 1 ) then
echo Note: not resubmitting job.
else
date
