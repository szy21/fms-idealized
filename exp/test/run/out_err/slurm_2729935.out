/central/home/zhaoyi/fms-idealized/exp/test/run
/home/zhaoyi/fms-idealized/idealized
num_segments = 10
*** Running /home/zhaoyi/fms-idealized/exp/test/run/run_test_1x_hpc on hpc-22-14 ***
Thu Oct 18 15:25:09 PDT 2018
MPI Used: /central/slurm/install/current/bin//srun
Using the following sourcecode modifications:
 Makefile is ready.
make: Nothing to be done for `all'.
NOTE from PE    0: MPP_DOMAINS_SET_STACK_SIZE: stack size set to    32768.
NOTE from PE    0: MPP_IO_SET_STACK_SIZE: stack size set to   131072.
NOTE from PE    0: MPP_DOMAINS_SET_STACK_SIZE: stack size set to   600000.

WARNING from PE    0: Namelist mix_snapshot_average_fields = true will cause ERROR in time coordinates of all time_averaged fields. Strongly recommend mix_snapshot_average_fields = false

  
ATMOS MODEL DOMAIN DECOMPOSITION
  X-AXIS =  128
  Y-AXIS =    8   8   8   8   8   8   8   8
mean surface pressure=1000.0000 mb

WARNING from PE    0: register_static_field: module/field dynamics/pk is STATIC, can NOT time ops


WARNING from PE    0: register_static_field: module/field dynamics/bk is STATIC, can NOT time ops


WARNING from PE    0: register_static_field: module/field dynamics/zsurf is STATIC, can NOT time ops


WARNING from PE    0: mixed_layer: mixed_layer restart file not found

 INSOLATION: running in perpetual equinox
 Integration completed through     1 days     0 seconds
 Integration completed through     2 days     0 seconds
 Integration completed through     3 days     0 seconds
 Integration completed through     4 days     0 seconds
 Integration completed through     5 days     0 seconds
 Integration completed through     6 days     0 seconds
 Integration completed through     7 days     0 seconds
 Integration completed through     8 days     0 seconds
 Integration completed through     9 days     0 seconds
 Integration completed through    10 days     0 seconds
NOTE from PE    0: MPP_IO_SET_STACK_SIZE: stack size set to   163840.
 MPP_DOMAINS_STACK high water mark=       23220

Tabulating mpp_clock statistics across    8 PEs...

                                          tmin          tmax          tavg          tstd  tfrac grain pemin pemax
Total runtime                        25.573850     25.573879     25.573858      0.000011  1.000     0     0     7
 MPP_STACK high water mark=           0
Completed run 1 of 1 in bsub 1.
WARNING: Existing analysis directory /central/groups/esm/zhaoyi//fms_tmp/test/test_1x_hpc/analysis/day0010h00 removed.
Submitted batch job 2729936
Note: not resubmitting job.
Thu Oct 18 15:25:37 PDT 2018
